lib/liquid/render.ex

```diff
@@ -7,8 +7,14 @@ defmodule Liquid.Render do
   alias Liquid.Tag
 
   def render(%Template{root: root}, %Context{}=context) do
-    { output, context } = render([], root, context)
-    { :ok, output |> List.flatten |> Enum.reverse |> Enum.join, context }
+    try do
+      { output, context } = render([], root, context)
+      { :ok, output |> List.flatten |> Enum.reverse |> Enum.join, context }
+    rescue
+      # TODO [future]: replace :ok with :error
```

Comment:
This seems to me a bit too global I would expect the error catching to be on the variable that is being parsed.

Because if there is a template

template = Liquid.Template.parse("Some other stuff {{ "value" | strip: wrong, number, of, arguments }}")
{ :ok, result, _ } = Liquid.Template.render(template, %{})

result = "Some other stuff Liquid error: Wrong number of arguments 4 of 1"